### DISCUSSION TITLE
Why is particular user appearing in error messages?

### DISCUSSION THREAD

**Discussion by cbowdenrm** [SafeExamBrowser/seb-win-refactoring#661] [Q&A]
![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/144977884/630776fd-3add-41fd-bfca-4f5672baf857)

Using seb 3.3.2

In the above image shows an error message of mention the suers directory of one of our staff members. 
His only involvement with regards to SEB is the signing of it with DigitCert. He appears in numerous other error messages.

Are these actual errors that would cause failures within SEB (because on testers/learners you not have that user JOHNDOE on their machines)?

How can we fix this error (which I assume is trying to clear the cache of a non-existing user)

**Comment by dbuechel** [comment]
No, that is part of the stack trace of an exception, specifically the path to the source file where the exception occurred. The reason why your staff member's name appears there is because he compiled SEB using his user account and thus the stack trace refers to the source file located under that user account. This has nothing to do nor can it cause issues when running on students machines.

**Reply by cbowdenrm** [reply]
Hi Damian 

Thank you for taking the time and effort to respond. I really appreciate it.

Your comments mirror what I observed yesterday in a session that he is compiling it on his machine.

Is there a way to prevent this/suppress it? Just looking to reduce the noise in the log file and eliminate future red-herrings and questions for such errors.

**Reply by danschlet** [reply]
Not to interfere with technical answers from @dbuechel, but I just wanted to clarify that we are not able to answer questions related to building your own/customized version of Safe Exam Browser. We can only answer such questions to [contributors of the SEB Alliance](https://safeexambrowser.org/alliance/fees.html) (requires Gold contributor level for consulting about SEB integrations and Platinum contributor level for consulting about customizing SEB). Thank you for your understanding!

**Reply by cbowdenrm** [reply]
Hi Daniel and Damian 

I understand where you are coming from; honestly, the assistance already provided has been invaluable. The important questions I needed have been answered. I realize I may be pushing the boundaries and acknowledge them.
